client/src/components/Authorization/Authorization.jsx

```diff
@@ -1,23 +1,32 @@
-import { useEffect, useState } from 'react';
-import { useLocation } from 'react-router';
+import { useEffect, useState  } from 'react'; 
+import { useLocation } from 'react-router-dom';
 import Login from '../../components/Authorization/Login/Login';
 import Registration from '../../components/Authorization/Registration/Registration';
+import styles from '../Authorization/Authorization.module.scss';
+import CLogo from '../../assets/images/MainPageImages/C.png';
 
 const Authorization = () => {
+  
   const location = useLocation();
+  
   const [pathName, setPathName] = useState(location.pathname);
-
   useEffect(() => {
     setPathName(location.pathname);
   }, [location]);
 
-  return (
-    <div>
-      Authorization
-      {pathName === '/login' ? <Login /> : ''}
-      {pathName === '/registration' ? <Registration /> : ''}
+  return(
+    <div className={styles.authorization}>
+      <div className={styles.authorization__title}>
+        <h1>Synergy</h1>
+        <p>Welcome to your project manager</p>
+      </div>
+      {pathName === '/login' ? <Login/> : ''}
```

Comment:
You can use && for logic operations
https://uk.reactjs.org/docs/conditional-rendering.html#inline-if-with-logical--operator